zcash_client_backend/src/data_api.rs

```diff
@@ -2114,6 +2117,11 @@ impl AccountBirthday {
         self.recover_until
     }
 
+    /// Returns the [`ChainState`] corresponding to the last block prior to the wallet's birthday
+    pub fn prior_chain_state(&self) -> &ChainState {
```

Comment:
Why does this need to be exposed?

If kept: addition should be documented in the changelog.